GDJS/Runtime/pixi-renderers/pixi-image-manager.ts

```diff
@@ -216,18 +216,30 @@ namespace gdjs {
      */
     getThreeMaterial(
       resourceName: string,
-      { useTransparentTexture }: { useTransparentTexture: boolean }
+      {
+        useTransparentTexture,
+        hasNoLighting,
+      }: { useTransparentTexture: boolean; hasNoLighting: boolean }
```

Comment:
Actually, can we name this `forceBasicMaterial`? To be 100% sure this is not confusing/aligned with the naming of 3D rendering engines (some could argue that basic material works with lighting, you just have to "bake" into your texture something that looks like shadows).

"hasNoLighting" could be confused with something that is related to the texture.